spinegeneric/cli/generate_figure.py

```diff
@@ -38,41 +39,6 @@
 hdlr = logging.StreamHandler(sys.stdout)
 logging.root.addHandler(hdlr)
 
-# List subject to remove, associated with contrast
-SUBJECTS_TO_REMOVE = [
-    # CSA
-    {'subject': 'sub-oxfordFmrib04', 'metric': 'csa_t1'},  # T1w scan is not aligned with other contrasts (subject repositioning)
```

Comment:
we need to keep the comments (ie: reason for removing the subject).
fortunately, yaml supports comments, so these should be ported to the new yaml files.
@valosekj would you mind generating this yaml file and i will upload it along with the database? thanks 🙏

Reply:
Sure 👍 I have created this file with comments and opened PR - https://github.com/spine-generic/data-multi-subject/pull/11